### DISCUSSION TITLE
Chatwoot-Slack integration only one-way

### DISCUSSION THREAD

**Discussion by admin-smsp** [chatwoot/chatwoot#5564] [Self hosted] [answered]
Hi Chatwoot Team,

I am trying to integrate Chatwoot with Slack with the Chatwoot documentation as a guide. But I'm stuck. I can't reply from the Slack thread. It's just not showing up in the Chatwoot and also to the Customer as well. But all the conversation from Chatwoot is sync to the Slack.

As you can see in the picture below. I've replied to the thread
<img width="360" alt="Screen Shot 2022-10-06 at 10 01 36 AM" src="https://user-images.githubusercontent.com/62686959/194204607-c2cdb2de-1fd9-4587-9770-98d9b3d9244b.png">

But it's not showing up here.
<img width="935" alt="Screen Shot 2022-10-06 at 10 00 34 AM" src="https://user-images.githubusercontent.com/62686959/194204634-513d2e8a-7717-46e9-80b0-babd41d5c251.png">

And also to the front-end.
![image (4)](https://user-images.githubusercontent.com/62686959/194208024-ee1d243f-f491-4615-93aa-1945197be765.png)

Eventhough my email in slack and Chatwoot is the same
<img width="360" alt="Screen Shot 2022-10-06 at 10 02 01 AM" src="https://user-images.githubusercontent.com/62686959/194208103-11248955-07bb-46d9-900e-66172405e7ab.png">

<img width="1982" alt="Screen Shot 2022-10-06 at 10 02 46 AM" src="https://user-images.githubusercontent.com/62686959/194208124-19d42d7e-75c8-4b4f-9b15-ed0a6ce8656a.png">

The agent email in slack and in Chatwoot is the same as the documentation said.

Here is what I've done:
1. Put the client ID and client secret from slack app to the Chatwoot .env file.
2. Put the redirect URL in the Slack app.
<img width="716" alt="Screen Shot 2022-10-06 at 9 07 37 AM" src="https://user-images.githubusercontent.com/62686959/194205151-8df2168c-7766-46f9-9d8c-8eb1d96b222a.png">
3. Bot token scopes
<img width="468" alt="Screen Shot 2022-10-06 at 10 21 42 AM" src="https://user-images.githubusercontent.com/62686959/194207768-9e8b36ac-d305-4396-bb87-b51f72ca374f.png">
4. User token scopes
<img width="700" alt="Screen Shot 2022-10-06 at 10 22 51 AM" src="https://user-images.githubusercontent.com/62686959/194207798-62b25a5d-68cd-4dde-bc58-a9c244660fe7.png">
5. Events and request URL
<img width="730" alt="Screen Shot 2022-10-06 at 9 08 08 AM" src="https://user-images.githubusercontent.com/62686959/194207604-d18f50d5-e343-4a89-9ed3-dbd131924783.png">
6. Subscribe bot events
<img width="693" alt="Screen Shot 2022-10-06 at 10 23 41 AM" src="https://user-images.githubusercontent.com/62686959/194207637-79dccde4-9c04-410e-8917-29e92c97082f.png">

Is there something that I miss to do it?
I follow the guide from https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/slack-integration-setup/ and https://www.chatwoot.com/docs/product/features/slack/

Thanks,

Teddy

**Comment by pranavrajs** [comment, marked as answer]
> ![image](https://user-images.githubusercontent.com/2246121/194209768-e6f56bff-1722-47dd-b6a5-5b8d487c38c5.png)

@admin-smsp Can you please try enabling the events here?
